packages/components/src/image/test/__snapshots__/index.native.js.snap

```diff
@@ -0,0 +1,58 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Image renders an img tag passing any additional props 1`] = `
+<View
+  style={
+    Array [
+      Object {
+        "aspectRatio": undefined,
+        "flex": 1,
+      },
+      undefined,
+    ]
+  }
+>
+  <Image
+    accessibilityLabel="WordPress logo"
+    data-id="123"
+    source={
+      Object {
+        "uri": "https://s.w.org/style/images/about/WordPress-logotype-wmark.png",
+      }
+    }
+    style={
+      Object {
+        "flex": 1,
+      }
+    }
+  />
+</View>
+`;
+
+exports[`Image renders an img tag with src and alt 1`] = `
+<View
+  style={
+    Array [
+      Object {
+        "aspectRatio": undefined,
+        "flex": 1,
+      },
+      undefined,
+    ]
+  }
+>
+  <Image
```

Comment:
Took me some time to convince myself that the `Image` here is the RN component, not the Image component we ourselves have defined in the components package. It's unfortunate that the names match like that 😬 . Nothing to change here, just wanted to share my "feelings" :).